.vib/harbor-notary-signer/goss/harbor-notary-signer.yaml

```diff
@@ -0,0 +1,16 @@
+group:
+  harbor:
+    exists: true
+user:
+  harbor:
+    exists: true
+file:
+  /etc/notary:
+    exists: true
+    mode: "0775"
+    owner: harbor
+    filetype: directory
+command:
+  check-app-run:
+    exec: notary-signer --help
+    exit-status: 2
```

Comment:
Please replace the help test with a version test.

Version can be displayed by running the `notary-signer` command without flags, although exit-status will return 1.

```
I have no name!@0c2cf4a39717:/vib$ notary-signer       
{"level":"info","msg":"Version: 0.6.1, Git commit: d6e1431f","time":"2023-04-25T09:42:31Z"}
{"level":"fatal","msg":"Could not read config at :, viper error: Unsupported Config Type \"\"","time":"2023-04-25T09:42:31Z"}
I have no name!@0c2cf4a39717:/vib$ echo $?
1
```

Reply:
When executing the notary-signer command, we can see that the version that is being displayed isn't the same version as the version of the application.
```
HOSTNAME=a2c3cd4dfab6
PWD=/tmp
OS_FLAVOUR=debian-11
HOME=/
SERVICE_NAME=notary_signer
TERM=xterm
SHLVL=1
BITNAMI_APP_NAME=harbor-notary-signer
APP_VERSION=2.8.0
OS_NAME=linux
PATH=/opt/bitnami/harbor-notary-signer/bin:/opt/bitnami/common/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
OS_ARCH=amd64
_=/usr/bin/env
OLDPWD=/tmp/bitnami

root@a2c3cd4dfab6:/tmp# notary-signer
{"level":"info","msg":"Version: 0.6.1, Git commit: d6e1431f","time":"2023-04-27T11:49:56Z"}
{"level":"fatal","msg":"Could not read config at :, viper error: Unsupported Config Type \"\"","time":"2023-04-27T11:49:56Z"}
```